src/bp/ui-studio/src/web/views/OneFlow/sidePanel/TopicList/index.tsx

```diff
@@ -2,6 +2,7 @@ import { AnchorButton, Button, Intent, Menu, MenuDivider, MenuItem, Position, To
 import axios from 'axios'
 import { Flow, Topic } from 'botpress/sdk'
 import { confirmDialog, lang, TreeView } from 'botpress/shared'
+import classNames from 'classnames'
```

Comment:
To keep the code clean (and styling), we try to use the shorthand "cx" instead specifically for that library, ex: `import cx from 'classnames'`